File: verificar-banco.js
const sqlite3 = require('sqlite3').verbose();

const db = new sqlite3.Database('./taxmaster.db', (err) => {
    if (err) {
        console.error('❌ Erro:', err);
        process.exit(1);
    }
});

// Mostrar estrutura da tabela
db.get("SELECT sql FROM sqlite_master WHERE type='table' AND name='processos'", (err, row) => {
    if (err || !row) {
        console.log('⚠️ Tabela processos não existe!');
    } else {
        console.log('📊 ESTRUTURA ATUAL:\n');
        console.log(row.sql);
        console.log('\n');
    }
    
    // Mostrar colunas
    db.all("PRAGMA table_info(processos)", (err, rows) => {
        if (!err && rows) {
            console.log('📋 COLUNAS EXISTENTES:\n');
            rows.forEach(col => {
                console.log(`   • ${col.name} (${col.type})`);
            });
            console.log('\n');
        }
        db.close();
    });
});

File: verificar_banco.py
import sqlite3

conn = sqlite3.connect('taxmaster.db')
cursor = conn.cursor()

# Pegar estrutura da tabela processos
cursor.execute("PRAGMA table_info(processos)")
colunas = cursor.fetchall()

print('\n📊 COLUNAS DA TABELA PROCESSOS:')
print('=' * 60)
for col in colunas:
    print(f'   {col[0]:2d}. {col[1]:30s} | {col[2]:10s} | NULL: {col[3]}')

print('\n✅ Total de colunas:', len(colunas))

# Verificar se numero_processo existe
tem_numero_processo = any(col[1] == 'numero_processo' for col in colunas)
print('\n🔍 Coluna "numero_processo" existe?', '✅ SIM' if tem_numero_processo else '❌ NÃO')

conn.close()
